### ISSUE TITLE
panic:Cannot read properties of undefined (reading 'replace')

### ISSUE THREAD

**Issue by shedali** [teaxyz/cli#424]
`tea -SE`

```README.md
  # Dependencies

  | Project                  | Version  |
  | ------------------------ | -------- |
_ | nodejs.org               | ^19.3.0  |
  | xcfile.dev               | ^0.0.111 |
  | +github.com/junegunn/fzf | ^0.38.0  |
```

```
----------------------------------------------------->> attachment begin
TypeError: Cannot read properties of undefined (reading 'replace')
    at file:///opt/tea.xyz/src-v0.24.8/src/hooks/useMoustaches.ts:73:50
    at Array.reduce (<anonymous>)
    at Object.apply (file:///opt/tea.xyz/src-v0.24.8/src/hooks/useMoustaches.ts:73:16)
    at fix (file:///opt/tea.xyz/src-v0.24.8/src/hooks/useVirtualEnv.ts:62:27)
    at default (file:///opt/tea.xyz/src-v0.24.8/src/hooks/useVirtualEnv.ts:43:20)
    at async file:///opt/tea.xyz/src-v0.24.8/src/app.ts:34:30
<<----------------------------------------------------- attachment end
error
```

**Comment by mxcl** [comment]
Can’t reproduce, what does `tea --version` say?

**Comment by mxcl** [comment]
K looks like it’s 0.24.8

**Comment by mxcl** [comment]
So I committed some defensive programming for this situation, but this may expose another bug since we are seemingly getting `undefined` when parsing Environment Variables from some file.

**Comment by jhheider** [comment]
I believe it's this:
```sh
% tea -E
TEA_PANTRY_PATH=/Users/jacob/tea/pantry.core:/Users/jacob/tea/pantry.extra
...
% echo -n "\n" >>.envrc
% tea -E               

panic: spilt tea. we’re sorry and we’ll fix it… but you have to report the bug!

    https://github.com/teaxyz/cli/issues/new?title=panic%3ACannot+read+properties+of+undefined+(reading+'replace')

----------------------------------------------------->> attachment begin
TypeError: Cannot read properties of undefined (reading 'replace')
    at file:///private/tmp/tea-cli-20230301-49246-gyq1g8/tea-0.24.7/src/hooks/useMoustaches.ts:73:50
    at Array.reduce (<anonymous>)
    at Object.apply (file:///private/tmp/tea-cli-20230301-49246-gyq1g8/tea-0.24.7/src/hooks/useMoustaches.ts:73:16)
    at fix (file:///private/tmp/tea-cli-20230301-49246-gyq1g8/tea-0.24.7/src/hooks/useVirtualEnv.ts:62:27)
    at default (file:///private/tmp/tea-cli-20230301-49246-gyq1g8/tea-0.24.7/src/hooks/useVirtualEnv.ts:43:20)
    at async file:///private/tmp/tea-cli-20230301-49246-gyq1g8/tea-0.24.7/src/app.ts:34:30
<<----------------------------------------------------- attachment end
```

I had blank lines in an existing `.envrc` that were used to group keys logically.

**Comment by jhheider** [comment]
Which, yes, the fix for 0.24.9 catches.
